.ci/Jenkinsfile

```diff
@@ -219,15 +219,15 @@ pipeline {
         axes {
           axis {
             name 'K8S_VERSION'
-            values "v1.23.0", "v1.22.0", "v1.21.1", "v1.20.7", "v1.19.11", "v1.18.19"
+            values "v1.24.0, v1.23.6, v1.22.9, v1.21.12"
```

Comment:
This should be 

```
            values "v1.24.0", "v1.23.6", "v1.22.9", "v1.21.12"
```

Reply:
fixed in https://github.com/elastic/elastic-agent/pull/560

Reply:
👍🏼 Thanks for catching this! I wonder how this PR passed the CI then 🤔

Reply:
I think we should include the `Jenkinsfile` I will do a pr for that, if not this can take multiple day to catch.

Reply:
https://github.com/elastic/elastic-agent/pull/568